cores/arduino/stm32/analog.c

```diff
@@ -614,7 +614,7 @@ uint16_t adc_read_value(PinName pin)
     return 0;
   }
 
-#if defined (STM32F0xx) || defined (STM32F1xx) || defined (STM32F3xx) || defined (STM32L4xx)
+#if defined (STM32F0xx) || defined (STM32F1xx) || defined (STM32F3xx) || defined (STM32L4xx) || defined (STM32L0xx)
```

Comment:
please invert L4 and L0 to be aligned with other #if test.
Alphabetical and numbering order.
F0/1/2/3/4/7- L0/1/4